site/en/docs/extensions/mv3/devtools/index.md

```diff
@@ -349,13 +352,15 @@ backgroundPageConnection.postMessage({
 ### Messaging from injected scripts to the DevTools page {: #evaluated-scripts-to-devtools }
 
 While the above solution works for content scripts, code that is injected directly into the page
-(e.g. through appending a `<script>` tag or through [`inspectedWindow.eval`][31]) requires a
-different strategy. In this context, [`runtime.sendMessage`][32] will not pass messages to the
-background script as expected.
+(e.g. through appending a `<script>` tag or through
+[`inspectedWindow.eval`][api-inspectedwindow-eval]) requires a different strategy. In this context,
+[`runtime.sendMessage`][api-runtime-sendmessage] will not pass messages to the background script as
+expected.
 
 As a workaround, you can combine your injected script with a content script that acts as an
-intermediary. To pass messages to the content script, you can use the [`window.postMessage`][33]
-API. Here's an example, assuming the background script from the previous section:
+intermediary. To pass messages to the content script, you can use the
+[`window.postMessage`][mdn-postmessage] API. Here's an example, assuming the background script from
```

Comment:
```suggestion
[`window.postMessage()`][mdn-postmessage] method. Here's an example, assuming the background script from
```

Reply:
API > method.

There's a best practice in this suggestion. Prefer the more specific term.